core/src/main/res/drawable/ic_filter_close_light.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
```

Comment:
I am not 100% sure but I think that this file does not need to be copied two times. When using an `attr` for the color, it can support light and dark theme with only one drawable file:

- `ic_filter_close` includes with attr `?attr/filter_close_color`
  - `@color/filter_close_light` is a selector depending on state
    - `@color/dialog_filter_button_active_light`
    - `@color/dialog_filter_button_inactive_light`
  - `@color/filter_close_dark` is a selector depending on state
    - `@color/dialog_filter_button_active_dark`
    - `@color/dialog_filter_button_inactive_dark`

I hope that it is clear what I mean. This would reduce code duplication of a rather complex file.

Reply:
👍 100% correct